related_website_sets.JSON

```diff
@@ -926,24 +926,6 @@
     }
   },
   {
-  	"contact": "david.molloy@bauermedia.co.uk",
-  	"primary": "https://graziadaily.co.uk",
-  	"associatedSites": [
-  		"https://heatworld.com",
-  		"https://closeronline.co.uk",
-  		"https://yours.co.uk",
-  		"https://motherandbaby.com",
-  		"https://takeabreak.co.uk"
-  	],
-  	"rationaleBySite": {
-  		"https://heatworld.com": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.",
-  		"https://closeronline.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.",
-  		"https://yours.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.",
-  		"https://motherandbaby.com": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.",
-  		"https://takeabreak.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains."
-  	}
-   },
-   {
```

Comment:
"contact": "david.molloy@bauermedia.co.uk", 
           "primary": "https://graziadaily.co.uk", 
           "associatedSites": [ 
                   "https://heatworld.com", 
                   "https://closeronline.co.uk", 
                   "https://yours.co.uk", 
                   "https://motherandbaby.com", 
                   "https://takeabreak.co.uk" 
           ], 
           "rationaleBySite": { 
                   "https://heatworld.com": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.", 
                   "https://closeronline.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.", 
                   "https://yours.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.", 
                   "https://motherandbaby.com": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.", 
                   "https://takeabreak.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains." 
           } 
    }, 
    {

Reply:
>           "contact": "david.molloy@bauermedia.co.uk", 
>            "primary": "https://graziadaily.co.uk", 
>            "associatedSites": [ 
>                    "https://heatworld.com", 
>                    "https://closeronline.co.uk", 
>                    "https://yours.co.uk", 
>                    "https://motherandbaby.com", 
>                    "https://takeabreak.co.uk" 
>            ], 
>            "rationaleBySite": { 
>                    "https://heatworld.com": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.", 
>                    "https://closeronline.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.", 
>                    "https://yours.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.", 
>                    "https://motherandbaby.com": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains.", 
>                    "https://takeabreak.co.uk": "Our websites are uniformly owned and operated by Bauer Media, a fact clearly indicated by the consistent footer across all domains." 
>            } 
>     }, 
>     {
> This comment was mark as resolved